app/cells/datagrid_filter_cell.rb

```diff
@@ -0,0 +1,24 @@
+class DatagridFilterCell < FormCellBase
+  def show
+    case options[:filter].type
+    when :enum
+      cell(
+        :select,
+        nil,
+        select_options
+      ).call(:show)
```

Comment:
Thought about that as well, but I hesitated since the external interface is defined in a view helper. Wasn't sure if it's really a good approach to call the view helper from the cell.